content/SCALE/SCALETutorials/Storage/Disks/ReplacingDisks.md

```diff
@@ -103,6 +103,13 @@ Select the new drive from the **Member Disk** dropdown list on the **Replacing d
 
    {{< trueimage src="/images/SCALE/Storage/ReplacingDiskDialog.png" alt="Replacing Disk Dialog" id="Replacing Disk Dialog" >}}
 
+
+**Force** overrides the safety check and adds the disk to the pool. Selecting this option erases any data stored on the disk!
+
+**Preserve Power Management and S.M.A.R.T. settings** transfers all power management and S.M.A.R.T. test configurations from the original disk to your replacement disk. The dialog window automatically selects this option, but you can uncheck it if you wish to reset your configurations.
```

Comment:
Style guide says to use "select to clear the option" and not "uncheck". How about this instead?
```suggestion
**Preserve Power Management and S.M.A.R.T. settings** transfers all power management and S.M.A.R.T. test configurations from the original disk to your replacement disk. This option is enabled by default. Select to clear the checkmark so you can reset your configurations.
```